src/ParticleSpecies.cpp

```diff
@@ -109,6 +109,17 @@ ParticleSpecies::flush(std::string const& path)
             patch.second.flush(patch.first);
     } else
     {
+        iterator it = find("position");
+        if ( it != end() )
+        {
```

Comment:
you can actually spare the `{}` for a single command :)

Reply:
I need to admit that I don't like that style that much, but since it's part of the [20 commandments](https://github.com/ComputationalRadiationPhysics/contributing/blob/master/codingGuideLines/cpp.md), I will comply ;)